### ISSUE TITLE
Add Dark and Light "Professional" Themes

### ISSUE THREAD

**Issue by GeorgeBerdovskiy** [PureChart/purechart#3]
## Background
While PureChart provides maximal flexibility when it comes to chart styling, we _do_ want to provide an abundance of default themes to choose from.

Following Ruby on Rails conventions, we've chosen to store chart styles in a specific folder. In the gem source code, they may be found under `lib/purechart/styles` as YML files. In a project using the gem, they must be inside of `app/purechart` and may be JSON, TOML, or YML / YAML files. Here's the default light theme as an example -

```yml
---
title:
  font: Inter Tight
  weight: 700
  color: "#000000"
labels:
  font: Inter Tight
  weight: 700
  color: "#000000"
ticks:
  font: Inter Tight
  weight: 400
  color: "#000000"
axes:
  style: "2px solid #000000"
gridlines:
  style: "2px dashed #00000033"
...
```

## Goal
Create a new theme called "professional" that uses a serif font like Times New Roman (although the choice is yours) and provides a palette of muted or pastel colors.
- Use [this](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html) page as a source on YAML syntax, including list definition
    - You'll need this because the colors should be defined as a list of dictionaries representing colors with two fields - name and hex value
- Visit [Google Fonts](https://fonts.google.com) for an excellent selection of free fonts!

**Comment by vivekbruh** [comment]
I'll work on this.

**Comment by viveknarayana** [comment]
I'll work on this

**Comment by GeorgeBerdovskiy** [comment]
Closed by #8
